src/app/hearing-part/components/hearings-table/hearings-table.component.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { Observable } from 'rxjs/Observable';
+import { CaseType } from '../../../core/reference/models/case-type';
+import { HearingType } from '../../../core/reference/models/hearing-type';
+import * as moment from 'moment';
+import { Priority, priorityValue } from '../../models/priority-model';
+import { HearingsTableComponent } from './hearings-table.component';
+import { DEFAULT_HEARING_FOR_LISTING_WITH_NOTES, HearingForListingWithNotes } from '../../models/hearing-for-listing-with-notes.model';
+
+const openDialogMockObjConfirmed = {
+    afterClosed: (): Observable<boolean> => Observable.of(true)
+};
+const now = moment();
+const openDialogMockObjDeclined = {
+    afterClosed: (): Observable<boolean> => Observable.of(false)
+};
+let matDialogSpy;
+let hearingService: any;
+let component: HearingsTableComponent;
+
+describe('HearingsTableComponent', () => {
+    beforeEach(() => {
+        matDialogSpy = jasmine.createSpyObj('MatDialog', ['open']);
+        hearingService = jasmine.createSpyObj('hearingService', ['deleteHearing']);
+        component = new HearingsTableComponent(matDialogSpy, hearingService);
+
+        component.hearings = [generateHearings('123')];
+    });
+
+    describe('Initial state ', () => {
+        it('should include priority', () => {
+            expect(component.hearings[0]).toEqual(generateHearings('123'));
```

Comment:
I guess *it* description is not related to its content. Can you adjust either test or title